shared/chat/inbox/container.js

```diff
@@ -47,6 +47,10 @@ const getSimpleRows = createSelector(
 
         return !isEmpty && !isSuperseded && !isFilteredOut
       })
+      .map(i => ({
```

Comment:
This probably warrants a comment explaining that this is done for perf reasons and that sorting immutable lists is slow, etc.

Reply:
sounds good